singer-io/tap-covid-19
# v.0.0.6 fix replication strategy

# Description of change
Fix replication strategy. Adjust `streams.py` and `sync.py` logic for `activate_version` based on source data replication strategy: single files vs. multiple files daily.

# Manual QA steps
Ran Discovery, initial sync, and incremental sync for all endpoints to target-stitch and singer-check-tap.
 
# Risks
Low - in Alpha testing
 
# Rollback steps
Revert to v.0.0.5
